### ISSUE TITLE
Can not change permission in sharing management panel (on CSS)

### ISSUE THREAD

**Issue by renyuneyun** [SolidOS/mashlib#149]
## Description

I set up a CSS instance and am using [the mashlib recipe](https://github.com/solid/community-server-recipes) to launch my service.

When trying to modify the (sharing) permission of the `profile` folder or anything inside it, nothing really works. I can not modify the permission -- removing myself from it, adding new person to it, drag-and-drop "global" icon to any row, and double-clicking "global" icon. None of them works.

## Error details

I can open the sharing panel (only the first time after refreshing the page; a separate issue), and can see the sharing target icons:
![图片](https://user-images.githubusercontent.com/3293723/153857315-f80c374c-c227-4153-93fc-30a0873a1923.png)

Double clicking "global" makes no effect. Drag-and-drop "global" also makes no effect.

Clicking on "person", "group" or "bot" allows me to input the URL to the agent, but then gives error:

```
Error: ACL file save failed: Web error: 401 (Unauthorized) on PUT of <https://solidpod.ryey.icu/ryey/profile/.acl>
```

## Additional contextual information

I found the issues with the permission and the sharing panel, as a result of trying to set my avatar. Here are some contextual information which might be relevant:

1.  Create a new user `USER` on my CSS instance `http://SERVER`, with its own namespace `http://SERVER/USER`.
2.  Upload a picture as the avatar.
3.  Refresh the page, finding out that the avatar icon on the web page is "broken image".
4.  Navigate to the avatar file `http://SERVER/USER/profile/AVATAR_FILE.jpg`, and found that it is only readable by myself.
5. Navigate to the `profile` folder containing the avatar, and found that it is also only readable by myself.
6. The `card` file is readable by everyone.

**Comment by bourgeoa** [comment]
Thanks for reporting. Already filed in https://github.com/solid/solidos/issues/113

Please do not file in mashlib but in SolidOS.

**Comment by bourgeoa** [comment]
> I set up a CSS instance and am using the mashlib recipe to launch my service.

I did not succeed to create/login in a CSS with mashlib recipe. How did you did it ?

**Comment by renyuneyun** [comment]
Thanks, @bourgeoa. I'll file new issues in solid/solidos .

I did not do anything special, but following the same instruction, and giving corresponding arguments to `npx community-solid-server` (especially `-b https://HOST.NAME`). The only noticeable difference to your case is that I'm doing HTTPS on a cloud server rather than HTTP on localhost (and I'm using nginx reverse proxy).
I also did not encounter the setup page. I presume this is because of the config in `config-mashlib.json` which gives these settings already...?


Of course, in my case, manual input of `https://SERVER/register/` is needed -- from "log in", "sign up" will result in `https://SERVER/register` which is not recognized. (Tracking this down made me find [CSS PR #1107](https://github.com/solid/community-server/pull/1107).)
